internal/utils/kruize/kruize_api.go

```diff
@@ -53,6 +53,16 @@ func Create_kruize_experiments(experiment_name string, k8s_object []map[string]i
 	if err := json.Unmarshal(body, &resdata); err != nil {
 		return nil, fmt.Errorf("can not unmarshal response data: %v", err)
 	}
+
+	// Temporary fix
+	// Currently, once Kruize pod inits it does not load performance-profile from DB
+	if strings.Contains(resdata["message"].(string), "Performance Profile doesn't exist") {
+		log.Error("Performance profile does not exist")
+		log.Info("Tring to create resource_optimization_openshift performance profile")
+		utils.Setup_kruize_performance_profile()
+		_, _ = Create_kruize_experiments(experiment_name, k8s_object)
```

Comment:
We need to return the response to main calling function. 

```suggestion
		if container_names, err := Create_kruize_experiments(experiment_name, k8s_object); err != nil {
				return nil, err
			} else {
				return container_names, nil
			}
```